guides/v2.3/frontend-dev-guide/templates/template-security.md

```diff
@@ -32,10 +32,19 @@ The following code sample illustrates the XSS-safe output in templates:
 <a href="<?php echo $block->escapeUrl($block->getUrl()) ?>"><?php echo $block->getAnchorTextHtml() ?></a>
 ```
 
-#### Escape functions for templates
+### Escape functions for templates
 
 For the following output cases, use the specified function to generate XSS-safe output.
 
+**Case:** Escape string for the CSS context
+**Function:** `escapeCss`
+
+```php
+<?php header('Content-Type: application/xhtml+xml; charset=UTF-8');?>
+<?php $css = "background: #000\"><script>alert(1)</script>"; ?>
+<div style="<?= $block->escapeCss($css) ?>"></div>
+```
```

Comment:
@jeff-matthews, please add here what in the output we will have the next:

```
<div style="background\3A \20 \23 000\22 \3E \3C script\3E alert\28 1\29 \3C \2F script\3E "></div>
```

Reply:
@atwixfirster, like this (https://github.com/magento/devdocs/pull/5299/commits/e9cc5658cdb1ee1745237860f430517f2e391652)?

Reply:
Nope, @jeff-matthews. I think we should not place the example code and the output in the same block.

I've expected to see something like that:

```php
<?php header('Content-Type: application/xhtml+xml; charset=UTF-8');?>
<?php $css = "background: #000\"><script>alert(1)</script>"; ?>
<div style="<?= $block->escapeCss($css) ?>"></div>
```

The escaping HTML will look like:

```
// Escaping CSS
<div style="background\3A \20 \23 000\22 \3E \3C script\3E alert\28 1\29 \3C \2F script\3E "></div>
```

Thank you!

Reply:
Got it @atwixfirster! I just pushed an update.